variables.tf

```diff
@@ -76,6 +77,16 @@ variable "tags" {
   description = "A map of tags to assign to all resources"
 }
 
+variable "permission_boundaries" {
+  type = object({
+    pipeline_boundary      = optional(string, null)
```

Comment:
Can you rename `pipeline_boundary` to `workspace_boundary`. This makes the terms the same accross the mcaf modules

Reply:
Done